src/appengine/requirements.txt

```diff
@@ -6,3 +6,6 @@ PyJWT==1.7.1
 pyOpenSSL==19.1.0
 requests-toolbelt==0.9.1
 sendgrid==6.0.4
+mock==4.0.2
```

Comment:
These are not required, already in Pipfile in clusterfuzz root.

Reply:
+1. Were you running into issues otherwise?

Reply:
I was, but I'll revert this commit and e-mail about them instead.